6/ansible-6.build

```diff
@@ -82,19 +82,19 @@ netapp.ontap: >=21.18.0,<22.0.0
 netapp.storagegrid: >=21.10.0,<22.0.0
 netapp.um_info: >=21.8.0,<22.0.0
 netapp_eseries.santricity: >=1.3.0,<2.0.0
-netbox.netbox: >=3.6.0,<4.0.0
+netbox.netbox: >=3.7.0,<4.0.0
 ngine_io.cloudstack: >=2.2.0,<3.0.0
 ngine_io.exoscale: >=1.0.0,<2.0.0
 ngine_io.vultr: >=1.1.0,<2.0.0
 openstack.cloud: >=1.8.0,<2.0.0
 openvswitch.openvswitch: >=2.1.0,<3.0.0
 ovirt.ovirt: >=2.0.0,<3.0.0
-purestorage.flasharray: >=1.12.0,<2.0.0
+purestorage.flasharray: >=1.13.0,<2.0.0
 purestorage.flashblade: >=1.9.0,<2.0.0
 sensu.sensu_go: >=1.13.0,<2.0.0
 servicenow.servicenow: >=1.0.0,<2.0.0
 splunk.es: >=1.0.0,<2.0.0
```

Comment:
```
[WARNING]: Collection splunk.es does not support Ansible version 2.13.0rc1
```

Reply:
splunk.es just released version 2.0.0: https://github.com/ansible-collections/splunk.es/releases/tag/2.0.0
Will be updated when I rebuild 6.0.0a2.